ext/standard/url.c

```diff
@@ -190,8 +190,8 @@ PHPAPI php_url *php_url_parse_ex(char const *str, int length)
 			memcpy(port_buf, p, (pp - p));
 			port_buf[pp - p] = '\0';
 			port = strtol(port_buf, NULL, 10);
-			if (port > 0 && port <= 65535) {
-				ret->port = (unsigned short) port;
+			if (port > 0 && port <= 4294967295) {
```

Comment:
This can't go into PHP 5.6 anyway, as it breaks the ABI (at least I think so -- the change of the port member size is absorbed by padding, but the values might not be consistent depending on machine endianness. It might be fine on x86, but will break on big endian machines.)